app/src/components/account/UserMenu.vue

```diff
@@ -39,9 +39,13 @@
           <div
             v-for="address in addresses"
             :key="
-              address.address.concat(
-                `-${address.networkId}-${address.sessionType}`
-              )
+              address.address
+                ? address.address.concat(
+                    `-${address.networkId}-${address.sessionType}`
+                  )
+                : address.name.concat(
```

Comment:
how can an account not have an address?

Reply:
probably due to my localStorage being broken for development reasons :shrug:  But still good to have to make our code safe